docs/data-ai/ai/run-inference.md

```diff
@@ -43,6 +43,10 @@ More powerful hardware, including the Jetson Xavier or Raspberry Pi 5 with an AI
 
 Vision services apply an ML model to a stream of images from a camera to generate bounding boxes or classifications.
 
+{{% alert title="Note" color="note" %}}
```

Comment:
Is this really relevant here? We're trying to cut down on boxes, and I'm thinking most/many readers of this section are working with 2D no? I'd think this would be more relevant in the context of pointclouds (perhaps in one or more module readmes? Haven't dug in this area but what do you think?)